common/src/main/java/com/wynntils/features/commands/CommandAliasesFeature.java

```diff
@@ -88,10 +72,5 @@ public boolean equals(Object other) {
             CommandAlias that = (CommandAlias) other;
             return originalCommand.equals(that.originalCommand) && aliases.equals(that.aliases);
         }
```

Comment:
Why did you keep the equals?

Reply:
Because they were custom made and not generated by intellij

Reply:
I don't think so? They seem to just compare the every field.